WarshipGame/Assets/Scripts/AI/Enemy.cs

```diff
@@ -133,6 +143,12 @@ private IEnumerator MoveTo()
             yield return null;
         }
 
+        MovementFinished();
+    }
+
+    private void MovementFinished()
+    {
+        
         _currentWaypoint = WaypointsScript.GetNextWaypoint(_currentWaypoint);
     }
```

Comment:
There is no reason for this one line to be it's own function as it is only used MoveTo so it can better be replaced with the code in it

Reply:
i wanted to work on a few things so i added it because i was already working in Enemy.cs.

its for an effect that plays when the enemy finishes. if you want i can take it out but i added it for some stuff im working on

Reply:
If you are working on it you should not push it to develop and push a version of this that is acceptable if it gets scrapped we will have a useless one line function